ctags/main/entry.c

```diff
@@ -366,8 +366,8 @@ extern void openTagFile (void)
 static int replacementTruncate (const char *const name, const long size)
 {
 	char *tempName = NULL;
-	FILE *fp = tempFile ("w", &tempName);
-	fclose (fp);
+	MIO *mio = tempFile ("w", &tempName);
+	fclose (mio);
```

Comment:
In addition there are ifdefs around and it gets compiled only when truncate() or ftruncate() are missing on the system (Geany's config.h defines HAVE_FTRUNCATE).